src/framework/recycledList/RecycledList.spec.jsx

```diff
@@ -113,6 +115,24 @@ describe('RecycledList', () => {
         expect(props.itemHeightProvider).toHaveBeenCalled();
       });
     });
+
+    describe('renderItem', () => {
+      it('is used to render the items', () => {
```

Comment:
Not really clear what is actually tested from the description. Perhaps just 'items', 'are rendered' or pass `renderItem` with default behaviour which would make the usage of the prop slightly more obvious!?